operations/app/src/environments/tf

```diff
@@ -172,7 +172,7 @@ if [[ "${ACTUAL_RESULTS?}" == "${EXPECTED_RESULTS?}" ]]; then
         ACTUAL_RESULTS="${ACTUAL_RESULTS?}${?}"
         ;;
 
-    import)
+    import*)
```

Comment:
why the `*` here?

Reply:
Because the command in use is `tf import <path> <id>`. Using just `import)` does not trigger this case.

Reply:
I think I get it, is it because we do: `./tf -e ${ENV_NAME} -c "import a b"`? And thus, it's `"import a b"` that we need to key off of, and not `"import"`.
That seams odd... could we not instead use `${*}` for the `${TFCMD_AFTER}` and take advantage of the `--` breaker on line 90 so that you do: `./tf -e "${ENV_NAME?}" -c import -- resource_type resource_name`?

So this would then turn into a
```bash
    import)
        # Some commands require arguments in a different location
        EXPECTED_RESULTS="${EXPECTED_RESULTS?}0"
        TFCMD_BEFORE=$(echo "${TFCMD?}" | cut -d' ' -f 1)
        terraform ${TFCMD_BEFORE?} \
            --var-file "${TFVARS?}" \
            ${*}
        ACTUAL_RESULTS="${ACTUAL_RESULTS?}${?}"
        ;;
```

Reply:
Ticketed as https://github.com/CDCgov/prime-reportstream/issues/1885